locales_dev/en/translation.json

```diff
@@ -85,9 +86,10 @@
   "Status": "Status",
   "Stops widget from refreshing": "Stops widget from refreshing",
   "Subheading": "Subheading",
-  "Submit Project": "Submit Project",
+  "Submit Project for Review": "Submit Project for Review",
   "Submit Task for Review": "Submit Task for Review",
   "Submit this task for review": "Submit this task for review",
+  "Submitting Project for Review…": "Submitting Project for Review…",
```

Comment:
We need to also manually apply these three changes to `locales/en/translation.json`.